### PR TITLE
Backend for Return Forms Component

### PR DESCRIPTION
**Pull request by vgoyal1996**

### CODE REVIEW

File: src/app/return-forms/return-forms.component.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    this.returnType = this.router.snapshot.paramMap.get('type');
+    this.apiService.getReturnFormsByReturnType(this.returnType)
+    .pipe(map(
+      res => {
+        return res.map(item => {
+          const resDate: Date = new Date(item.dueDateOfFiling);
+
+          return {
+            formName: item.formName,
+            dueDateOfFiling: this.datepipe.transform(resDate, 'MMM d, y'),
```
Comment (priya-bansal) [inline src/app/return-forms/return-forms.component.ts:152]: is this the correct date form ?

File: src/app/return-forms/return-forms.component.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        this.onNewReturnFormAdded(result);
+      }
+    });
+  }
+
+  onNewReturnFormAdded(newReturnFormValue: ReturnForm): void {
+    console.log("event triggered");
+    this.dataSource.push({
+      formName: newReturnFormValue.formName,
+      dueDateOfFiling: this.datepipe.transform(new Date(newReturnFormValue.dueDateOfFiling), 'MMM d, y'),
```
Comment (priya-bansal) [inline src/app/return-forms/return-forms.component.ts:135]: same here

File: src/app/return-forms/return-forms.component.ts (hunk 1 above)
Comment (vgoyal1996) [reply src/app/return-forms/return-forms.component.ts:152]: https://www.angularjswiki.com/angular/angular-date-pipe-formatting-date-times-in-angular-with-examples/

File: src/app/return-forms/return-forms.component.ts (hunk 2 above)
Comment (vgoyal1996) [reply src/app/return-forms/return-forms.component.ts:135]: same

**Pull request merged**